constants.py

```diff
@@ -88,8 +88,8 @@ def error_string(error, logname, with_hd):
 
 # filesystems and partitions types:
 bootfs_type = 'vfat'
-rootfs_type = 'ext3'
-logsfs_type = 'ext3'
+dom0fs_types = ('ext3', 'ext4')
+default_dom0fs_type = 'ext4'
```

Comment:
I think for compatibility the default type should be `ext3`.

Reply:
Why the rename `root` -> `dom0`? I mean, logs and boot are also `dom0` partitions. `root` was more clear to me, it's the partition to be mounted in root (`/`). If `rootfs_type` and `rootfs_types` were too similar I would use `allowed_rootfs_types`.

Reply:
It was to avoid confusion :D thanks for the suggestion, will update!
Part of the aim of this PR is to allow us to use EXT4 by default in Dom0 for XS9, so making the default EXT3 would nullify that effort